cpp/cpp/cpp/cpp.cpp

```
@@ -0,0 +1,32 @@
+// cpp.cpp : This file contains the 'main' function. Program execution begins and ends there.
+//
+
+#include <iostream>
+
+int main()
+{
+    int num{ };
+    double result{ };
+
+    std::cin >> num;
+    if (num >= 0) {
+        result = std::sqrt(num);
+        std::cout << result;
+    }
+    else {
+        std::cout << "Number can't be < 0\n";
```

You've successfully identified this as a bad input, why not return a proper error code (other than 0)?